userguide/workingwithproducts.rst

```diff
@@ -68,12 +68,12 @@ The Identifier is the unique way to identify each product. Every line in the CSV
 
     The identifier is the only mandatory attribute to import products.
 
-Example
+.. code-block:: none
```

Comment:
this page is not even displayed i the generated doc, at some point we discussed about put the user guide in the same repo to ease the contribution, traduction, etc. But our tech doc format is a bit too limited for user guide so we decided to keep them separated. Don't hesitate to drop these useless user guide pages.